support-frontend/assets/components/product/ProductInfoChip.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+
+type PropTypes = {|
+  children: Node,
+  icon?: Node
+|};
+
+const infoChip = css`
+  ${textSans.medium()}
+
+  &:not(:last-of-type) {
+    margin-bottom: ${space[4]}px;
+  }
+`;
+
+const infoChipIcon = css`
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  vertical-align: text-bottom;
+  width: ${space[6]}px;
+  height: ${space[6]}px;
+  margin-right: ${space[3]}px;
+
+  & > svg {
+    /* Repeat height here to fix Safari issue with inline-flex SVGs */
+    height: ${space[6]}px;
+    display: block;
+    fill: currentColor;
+  }
+
+  /* Hacky fix for our gift svg */
+  & svg.svg-gift {
+    height: unset;
+  }
```

Comment:
👏  - it's such a credit to your good browserstack habits that you've picked this up in advance of PR / QA testing.

Reply:
The benefit of screen reader testing is picking up the inevitable weird Safari problems 😄